jest.config.js

```diff
@@ -35,7 +35,7 @@ module.exports = {
   ],
   transform: {
     "^.+\\.js$": `<rootDir>/jest-transformer.js`,
-    "^.+\\.tsx?$": `<rootDir>/node_modules/ts-jest/preprocessor.js`,
+    "^.+\\.tsx?$": `<rootDir>/jest-transformer.js`,
```

Comment:
These could be combined into one regex. e.g. ` "^.+\\.[tj]sx?$"`

Reply:
I saw that in your PR! I'll leave this and let your PR make the better change :)